mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,717 @@
+# Copyright 2022 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+!ruby/object:Api::Product
+name: CloudRunV2
+display_name: Cloud Run (2nd gen)
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+versions:
+  - !ruby/object:Api::Product::Version
+    name: ga
+    base_url: https://run.googleapis.com/v2/
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Cloud Run API
+    url: https://console.cloud.google.com/apis/library/run.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: "Service"
+    base_url: projects/{{project}}/locations/{{location}}/services
+    self_link: projects/{{project}}/locations/{{location}}/services/{{name}}
+    create_url: projects/{{project}}/locations/{{location}}/services?serviceId={{name}}
+    update_verb: :PATCH
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/run/docs/'
+      api: "https://cloud.google.com/run/docs/reference/rest/v2/projects.locations.jobs"
```

Comment:
Should this be `https://cloud.google.com/run/docs/reference/rest/v2/projects.locations.services` instead?

Reply:
Good catch! Fixed!